code/modules/reagents/chemistry/reagents/drugs.dm

```diff
@@ -487,23 +551,32 @@
 	color = "#60A584" // rgb: 96, 165, 132
 	taste_description = "bitterness"
 
+/datum/reagent/aranesp/on_mob_add(mob/living/L)
+	if(ishuman(L))
+		var/mob/living/carbon/human/H = L
+		H.physiology.stun_mod *= 0.7
+		H.physiology.stamina_mod *= 0.7
+
 /datum/reagent/aranesp/on_mob_life(mob/living/M)
 	var/update_flags = STATUS_UPDATE_NONE
-	update_flags |= M.adjustStaminaLoss(-40, FALSE)
-	if(prob(90))
-		update_flags |= M.adjustToxLoss(1, FALSE)
+	update_flags |= M.adjustStaminaLoss(-2, FALSE) // small amount of stam regen, meaning that if you can buy time between hits you can take 6 disabler shots before going down
```

Comment:
Disabler is 30 stam; 6 disabler shots is 180 damage;
With base stam regeneration of 1 and bonus 2 from anaresp, you'd need to buy 80/3 = 27 life cycles, or 53 seconds between shots for the comment to be true...

Reply:
aranesp reduces the stam damage by 30%. 
so its 21 damage per shot
21 * 5 is 105
so its only 2 cycles or 4 seconds

Reply:
```suggestion
	// small amount of stam regen, meaning that if you can buy time between hits
	// you can take 6 disabler shots (30 base * 0.7 stam mod = 21 stam damage each) before going down
	update_flags |= M.adjustStaminaLoss(-2, FALSE) 
```